biojava-genome/src/main/java/org/biojava/nbio/genome/util/ChromosomeToProteinMapper.java

```diff
@@ -0,0 +1,193 @@
+package org.biojava.nbio.genome.util;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+import org.biojava.nbio.core.exceptions.CompoundNotFoundException;
+import org.biojava.nbio.core.sequence.DNASequence;
+import org.biojava.nbio.core.sequence.RNASequence;
+import org.biojava.nbio.core.sequence.compound.NucleotideCompound;
+import org.biojava.nbio.core.sequence.template.SequenceView;
+import org.biojava.nbio.genome.parsers.genename.GeneChromosomePosition;
+import org.biojava.nbio.genome.parsers.twobit.TwoBitParser;
+
+import com.google.common.collect.Range;
+
+public class ChromosomeToProteinMapper {
+	
+	private TwoBitParser parser;
+	private String GENOME_URI;
+	
+	/** Sets a path to the genome.2bit file.
+	 * 
+	 *  @param path The path to the file containing the genome in .2bit format.
+	 */
+	public void setGenomeURI(String path) {
+		GENOME_URI=path;
+	}
+	
+	/** 
+	 *  Reads a genome from a locally stored .2bit file.
+	 */
+	public void readGenome() throws Exception {
+		File f = new File(GENOME_URI);
+		this.parser = new TwoBitParser(f);
+	}
+	
+	/** Sets a chromosome for TwoBitParser.
+	 * 
+	 * @param chr The chromosome name (e.g. chr21)
+	 */
+	public void setChromosome(String chr) throws Exception {
+		parser.close();
+		String[] names = parser.getSequenceNames();
+		for(int i=0;i<names.length;i++) {
+			if ( names[i].equals(chr) ) {
+				parser.setCurrentSequence(names[i]);
+				break;
+			}
+		}
+	}
+	
+    /** Extracts the exons boundaries in CDS coordinates for genes living on the reverse DNA strand.
+     *
+     * @param exonStarts The list holding the genetic coordinates pointing to the start positions of the exons (including UTR regions)  
+     * @param exonEnds The list holding the genetic coordinates pointing to the end positions of the exons (including UTR regions)
+     * @param cdsStart The start position of a coding region
+     * @param cdsEnd The end position of a coding region
+     * 
+     * @return the list of genetic positions corresponding to the exons boundaries in CDS coordinates
+    */
+    public static List<Range<Integer>> getCDSRegionsReverse(List<Integer> exonStarts, List<Integer> exonEnds,
```

Comment:
can you move these methods to the ChromosomeMapping tools? There are already a set of utility methods like that there.